app/bundles/LeadBundle/Entity/Tag.php

```diff
@@ -75,11 +98,23 @@ public function getTag()
      * @param string $tag
      *
      * @return Tag
+     *
+     * @deprecated in 2.12.0. To be removed in 3.0.0. Use constructor instead
```

Comment:
+1

Reply:
Does this work with Symfony collection forms with Doctrine associations?